app/models/activity.rb

```diff
@@ -12,6 +12,11 @@ def time_formatted
     created_at.strftime("%-I:%M %p")
   end
 
+  def whodunnit
+    user_id = PaperTrail.whodunnit
```

Comment:
I cannot think of a scenario where PaperTrail.whodunnit would be `nil`. Can you? If so, I will add a conditional to deal with what to do when `nil`